docs/guides/admin/docs/upgrade.md

```diff
@@ -29,6 +29,7 @@ Or replace the old Java version like this:
 ```
 % dnf shell
 > remove java-1.8.0-openjdk.x86_64
+> remove java-1.8.0-openjdk-headless.x86_64
```

Comment:
To not forget yet another package, how about using:

```suggestion
> remove 'java-1.8.0*'
```

Reply:
I like it! Didn't know that was possible, thanks. :smile: